### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by b2dac** [OlegKi/jqGrid#5]
Please update your documentation so that it includes all the great new features that you have added.

**Comment by OlegKi** [comment]
it's in my TODO list, but I need **create** new documentation instead of **updating** an existing one. :-)

**Comment by fletchsod-developer** [comment]
It would be awesome to have that in GitHib's wiki instead of relying on web url somewhere.

A good example of what I meant is here.  :-)

    https://github.com/Microsoft/TypeScript/wiki

**Comment by OlegKi** [comment]
It's exactly what I planed before.

**Comment by OlegKi** [comment]
@fletchsod-developer : It's exactly what I planed before.

**Comment by AuthorProxy** [comment]
@fletchsod-developer I think wiki and common site should exist together, like a lot of services do. Because it will be nice if you will have some entry point, and some live examples of jqgrid as well, like on the existing not-free-jqgrid documentation.

**Comment by OlegKi** [comment]
I agree with Alex ( @AuthorProxy ). I want to create documentation, which describes in details all options, all methods and all callbacks/events of jqGrid.

I want to place on http://free-jqgrid.github.io/ the documentation + list of demos grouped by features. I want to use the wiki (https://github.com/free-jqgrid/jqGrid/wiki) as the place for more detailed explanation of some common problems and the ways of solving it. So I see requirement of existing both parts of "documentation". I posted for example some relatively simple [changes](https://github.com/free-jqgrid/jqGrid/commit/9d838631c6c1a3809b3e61483421a77ab1740e01) in `formatter: "actions"` which allows to include custom inline buttons. After publishing of new features I will post new wiki which will describes the feature in details.

There are two problems in creating of documentation:
 1. **time problem** in general. I can't invest all my time in developing of free product.
 2. **starting point** in creating of *the base web design* of http://free-jqgrid.github.io/. I'm not a designer. The structure of site which can be automatically generated (like http://free-jqgrid.github.io/jqGrid/) is not what I want to have. **It would be wonderful if some designer would suggest a prototype of the documentation site and I will fill it later with more information.** Exactly the starting point of the documentation is currently difficult for me, but I could sure find a little time to fill it with content in small steps if I would have a good starting point.

**Comment by fletchsod-developer** [comment]
I didn't say wiki & common site shouldn't exists together, just saying.  But the documentation at Tony's jqGrid is way out of date and cause me issues when I first time made one.

**Comment by mrjumpy** [comment]
@OlegKi 
Maybe I can help you to build some example?(I'm not designer)
just like you want. We'll build it inside http://free-jqgrid.github.io/
Otherwise, Developers always going to jqGrid official site all the time.

**Comment by OlegKi** [comment]
@mrjumpy: Thank you for your suggestion. You can post your suggestions as pull request to https://github.com/free-jqgrid/free-jqgrid.github.io. I started free jqGrid at the end of December 2014. The project takes very much time. I have to make additional projects for my customers to get enough money for me and my family too. So I'm really glad it somebody from the community would help to improve jqGrid or the documentation. One can post pull requests to https://github.com/free-jqgrid/free-jqgrid.github.io or to modify or to post new articles in [the wiki](https://github.com/free-jqgrid/jqGrid/wiki). The wiki is opened for writing for everybody. I trace the changes, but till now the most users prefer just to ask new requirements and uses the results instead of doing something for the project. So **every help is welcome!**

**Comment by opticyclic** [comment]
If I wanted to create a few sample projects (probably using Java and Jetty) where should I put them?
Should I create an examples sub-dir in this project and make a pull request or should it be in the other repo?

**Comment by OlegKi** [comment]
What would be better for you? I think that the exact place is not so important. You can create some examples with short or long descriptions in your own repository. I can create repository under https://github.com/free-jqgrid organization, where you can place the same code. I personally have created free jqGrid fork under my own repository and moved the repository under https://github.com/free-jqgrid organization later after I created the organization. So one can move personal repository under organization of GitHub in some clicks. The main problem which I see is in areas where one can directly post changes without any pull requests. Thus I see the most simple way would be creating separate repository where you publish all what you want. Look at the organisation unit https://github.com/jquery/ which contains many separate repositories. In the same way free-jqgrid organisation unit can hold many separate repositories. Nevertheless I would suggest you to post some content in your own repository at the beginning. We can move the content later in repository under free-jqgrid organisation unit and you would have direct write access in the repository after the movement. What do you think about such way?

**Comment by OlegKi** [comment]
One more small suggestion to you: you can write any new [wiki article](https://github.com/free-jqgrid/jqGrid/wiki), for example in usage of free jqGrid from Webjars or about using free jqGrid from Java or Jetty or about any other subject. You can modify existing article too. Currently everybody have write access to the wiki of free-jqgrid. I just review articles posted by other people (see [the article](https://github.com/free-jqgrid/jqGrid/wiki/OData-plugin-for-jqGrid) for example).

**Comment by mkyadav** [comment]
is there any good documentation besides trirand.com?

**Comment by OlegKi** [comment]
@mkyadav: Somebody have to write the good documentation before. One need to invest many time for that. What information you are searching for? Currently there are only [the wiki](https://github.com/free-jqgrid/jqGrid/wiki), [the starting point](http://free-jqgrid.github.io/getting-started/index.html) of the full documentation and a lot of old answers on [the stackoverflow](http://stackoverflow.com/questions/tagged/jqgrid%20free-jqgrid?mode=any).
